### PR TITLE
Tweaks the transformer

### LINKED ISSUE #2634: [Newmap] Mobs can be pulled out of robotic transformer while it is transforming them
The mob still transforms despite not being located inside the transformer.

### LINKED ISSUE #2635: [Newmap] Robotic transformer creates conveyors in nullspace
This causes runtimes every time the conveyor tries to process.

### PR DESCRIPTION
**Pull request by printer16**
Does some tweaking and fixing to the transformer.
Fixes #2634 
Fixes #2635

### CODE REVIEW

**Comment by NanakoAC** [comment]
huh? What is this transformer? Is this something that was added anew in the past month? I've never heard of it

**Comment by LordFowl** [comment]
wtf nanako

**Comment by NanakoAC** [comment]
oh shit oops, that was a misclick. I didnt intend to close this PR

**Comment by MarinaGryphon** [comment]
It's the Robotic Transformer thing. It's researchable by Malf AIs, borgifies people. Uses a conveyor basically. Human goes in, borg comes out.

**Comment by skull132** [comment]
@printer16 clarification of the PR is needed. Why is this what is this.

**Comment by printer16** [comment]
It prevents the conveyors from spawning in null space if the main machine doesn't have a location set for whatever reason which is what was causing the conveyors spawning in null space. If it doesn't have a location, give it 5 DS to finish initializing and spawn itself then make the conveyors.

File: code/game/machinery/transformer.dm [hunk 1]
```diff
@@ -13,7 +13,13 @@
...
 	// On us
 	. = ..()
-	new /obj/machinery/conveyor(loc, WEST, 1) // this doesnt need to check for a location because it is under the machine so its assumed there is one
+	if(loc)
+		MakeConveyor()
+	else
+		addtimer(CALLBACK(src, .proc/MakeConveyor), 5)
+
+/obj/machinery/transformer/proc/MakeConveyor()
+	new /obj/machinery/conveyor(loc, WEST, 1)
```
Comment (skull132) [inline code/game/machinery/transformer.dm]: Maybe add a safety here:
```
if (!loc)
    PROCLOG("Trying to spawn conveyor in null space.")
    return
```

Comment (skull132) [inline code/game/machinery/transformer.dm]: If this doesn't compile, use `PROCLOG_WEIRD`

**Pull request merged**